mage_ai/data_preparation/templates/data_exporters/bigquery.py

```diff
@@ -25,13 +26,14 @@ def export_data_to_big_query(df: DataFrame) -> None:
     ```
     BigQuery.with_credentials_object({'service_key': ...}, **kwargs)
     ```
+
+    Alternatively, all parameters can be specified in the configuration file.
     """
     table_id = 'your-project.your_dataset.your_table_name'
-    config = {
-        # Specify any other configuration settings here to pass to BigQuery client
-        'project': 'your_project_name',
-    }
-    return BigQuery(**config).export(
+    config_path = 'path/to/your/io/config/file.yaml'
```

Comment:
Let's set a default path here since we auto generated the io_config.yaml

Reply:
Updated default path to use the autogenerated config path in templates.